.github/workflows/roll-stable.yml

```diff
@@ -16,11 +16,11 @@ jobs:
         id: determine-version
         with:
           script: |
-            const allBranches = await github.rest.repos.listBranches({
+            const allBranches = await github.paginate("GET /repos/{owner}/{repo}/branches", {
```

Comment:
main fix here is that before it was paginated, so we were only getting the first 30 results. We want all.